### PR TITLE
Add a link to the Cypher manual and fix links to the APOC docs

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-getting-started-379.surge.sh

Updated pages:
- [Graph model refactoring](https://neo4j-docs-getting-started-379.surge.sh/getting-started/data-modeling/graph-model-refactoring)

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 1]
```diff
@@ -122,19 +120,22 @@ YIELD batches
...
   ** a list of values that should be considered `true`
   ** a list of values that should be considered `false`
 * Configuration values for the procedure, including:
-  ** `params` - parameters passed into those Cypher queries
-  ** `batchSize`- controls the number of inner statements that are run within a single transaction
+  ** `params` - parameters passed into those Cypher queries.
+  ** `batchSize`- controls the number of inner statements that are run within a single transaction.
 
-When we run this query we see the following output:
+After runnign the query, you see the following output:
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:138]:
```suggestion
After running the query, you see the following output:
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-getting-started-379.surge.sh

Updated pages:
- [Graph model refactoring](https://neo4j-docs-getting-started-379.surge.sh/getting-started/data-modeling/graph-model-refactoring)

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 2]
```diff
@@ -1,26 +1,25 @@
 [[graph-model-refactoring]]
 = Graph model refactoring
 :tags: graph-modeling, data-model, schema, refactoring, apoc
-:description: Building on the Cypher Basics guides, this guide provides a worked example of changing a graph model. Upon finishing this guide, you should be able to evolve your graph model based on changing requirements.
+:description: This guide provides a worked example of changing a graph model. Upon finishing this guide, you should be able to evolve your graph model based on changing requirements.
```
Comment (lidiazuin) [inline modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:4]: What does "worked" mean here? As in "working"?

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 3]
```diff
@@ -122,19 +120,22 @@ YIELD batches
...
 
-The `apoc.periodic.iterate` procedure in this query takes in three parameters:
+For more details about the `UNWIND` clause, go to the link:https://neo4j.com/docs/cypher-manual/current/clauses/unwind/[Cypher manual -> UNWIND].
+
+The `apoc.periodic.iterate` procedure in the above query takes in three parameters:
 
 * An outer Cypher query that finds and returns a stream of `CONNECTED_TO` relationships to be processed.
-* An inner Cypher query that processes those `CONNECTED_TO` relationships, converting to boolean any values for the specified property on those relationships. It does this using the `apoc.refactor.normalizeAsBoolean` procedure, which itself takes in several parameters:
+* An inner Cypher query that processes those `CONNECTED_TO` relationships, converting to boolean any values for the specified property on those relationships.
+It does this using the `apoc.refactor.normalizeAsBoolean` procedure, which itself takes in several parameters:
```
Comment (lidiazuin) [inline modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:129]:
```suggestion
It does this by using the `apoc.refactor.normalizeAsBoolean` procedure, which itself takes in several parameters:
```

**Review by lidiazuin — CHANGES_REQUESTED** [review]
thanks Natalia! I left some comments

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc (hunk 2 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:4]:
```suggestion
:description: This guide provides a working example of changing a graph model. Upon finishing this guide, you should be able to evolve your graph model based on changing requirements.
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:4]: yeah, I think, you're right. They mean 'working'

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 4]
```diff
@@ -122,19 +120,22 @@ YIELD batches
 RETURN propertyToDelete, batches
 ----
 
-The `apoc.periodic.iterate` procedure in this query takes in three parameters:
+For more details about the `UNWIND` clause, go to the link:https://neo4j.com/docs/cypher-manual/current/clauses/unwind/[Cypher manual -> UNWIND].
```
[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:123]: I prefer to keep 'The Cypher manual' in the link description.

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 5]
```diff
@@ -176,11 +177,11 @@ RETURN origin.code AS origin,
...
 [#create-node-from-relationship]
-== Create node from relationship
+== Create a node from a relationship
 
-Next, imagine that we want to write a query that finds a specific flight.
-This is quite difficult with our existing model because flights are represented as relationships.
-We can evolve our model to create a `Flight` node from the properties stored on the `CONNECTED_TO` relationship.
+Imagine that you want to write a query that finds a specific flight.
+This is quite difficult with our existing data model because flights are represented as relationships.
+You can evolve the model to create a `Flight` node from the properties stored on the `CONNECTED_TO` relationship.
```
[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:184]: I'm not sure that this is an improvement. I think, it's just a change to address your current needs

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-getting-started-379.surge.sh

Updated pages:
- [Graph model refactoring](https://neo4j-docs-getting-started-379.surge.sh/getting-started/data-modeling/graph-model-refactoring)

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 6]
```diff
@@ -1,26 +1,25 @@
 [[graph-model-refactoring]]
 = Graph model refactoring
 :tags: graph-modeling, data-model, schema, refactoring, apoc
-:description: Building on the Cypher Basics guides, this guide provides a worked example of changing a graph model. Upon finishing this guide, you should be able to evolve your graph model based on changing requirements.
+:description: This guide provides a working example of changing a graph model. Upon finishing this guide, you should be able to evolve your graph model based on changing requirements.
 
 == Introduction
 
 Building on the Cypher basic concepts, this guide provides a worked example of changing a graph model.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:8]:
```suggestion
Building on the Cypher basic concepts, this guide provides a working example of changing a graph model.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-getting-started-379.surge.sh

Updated pages:
- [Graph model refactoring](https://neo4j-docs-getting-started-379.surge.sh/getting-started/data-modeling/graph-model-refactoring)

File: modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc [hunk 7]
```diff
@@ -296,7 +299,7 @@ If we run this query we'll see the following output:
 | 1       | 1000  | 0         | 1000                | 0                | 0             | 0       | {}            | {total: 1, committed: 1, failed: 0, errors: {}} | {total: 1000, committed: 1000, failed: 0, errors: {}} | FALSE
 |===
 
-We can then write the following query to find the airlines and number of flights involving each:
+You can then write the following query to find the airlines and number of flights involving each:
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/data-modeling/graph-model-refactoring.adoc:302]:
```suggestion
You can then write the following query to find the airlines and number of flights involving each airline:
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-getting-started-379.surge.sh

Updated pages:
- [Graph model refactoring](https://neo4j-docs-getting-started-379.surge.sh/getting-started/data-modeling/graph-model-refactoring)

**Pull request merged**